packages/mobile/src/components/offline-downloads/DownloadToggle.tsx

```diff
@@ -24,9 +25,10 @@ export type TrackForDownload = {
 }
 
 type DownloadToggleProps = {
-  labelText?: string
   tracksForDownload: TrackForDownload[]
-  collection?: string
+  labelText?: string
+  collectionId?: number
+  isFavoritesDownlaod?: boolean
```

Comment:
downlaod

Reply:
🤣 If I see the word download one more time 😡